blockly-rtc/src/MarkerManager.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Wrapper class for interacting with the Blockly MarkerManager.
+ * @author navil@google.com (Navil Perez)
+ */
+
+import * as Blockly from 'blockly/dist';
+import MarkerUpdate from './MarkerUpdate';
+
+export default class MarkerManager {
+  constructor(workspaceId, sendMarkerUpdate, getMarkerUpdates,
+      getBroadcastMarkerUpdates) {
+    this.workspaceId = workspaceId
+    this.colours =  [
```

Comment:
nit: remove extra space after ``this.colours``.

Reply:
Done.